components/canvas.py

```diff
@@ -39,12 +45,9 @@ def DoSetViewport(self):
         width, height = size = self.size = self.GetClientSize()
         self.SetCurrent(self.context)
 
-        # set viewport dimensions to square (may change later)
-        min_size = min(width, height)
-        glViewport(0, 0, min_size, min_size)
+        glViewport(0, 0, width, height)
+        self.aspect_ratio = width / height;
```

Comment:
Can you make sure if min_size variable is in use? If not, please remove the variable and any commented out code using min_size.

Reply:
I've already removed the line containing min_size (shown in red), and there aren't any more instances of it